### ISSUE TITLE
Range selector scrolls away while scrollablePlotArea enabled

### ISSUE THREAD

**Issue by nurbek-mirai** [highcharts/highcharts#20940]
#### Expected behaviour
Range selector persists on top while scrolling(absolute/fixed positioning) like xAxis header

#### Actual behaviour
Range selector scrolls away like points

#### Live demo with steps to reproduce
https://jsfiddle.net/9esod03L/12/

#### Product version
Highcharts JS v11.4.0 (2024-03-04)
Highcharts Gantt JS v11.4.0 (2024-03-04)

#### Affected browser(s)
All Browsers

Also when you put `scrollablePlotArea.opacity ` to `1` the Range selector appears behind doesn't even show on the UI, So there is 2 bugs:
1. Range selector appears behind
2. Range selector is not persisted while scrolling

Sorry for my bad English.

**Comment by bm64** [comment]
Hello @nurbek-mirai,

I don't think it's a bug, as the Range Selector is part of Highcharts Stock and may not have been optimized for use with scrollablePlotArea.

However, this functionality could be a useful improvement so I'm labeling it as a feature request.

**Comment by pawelfus** [comment]
**Workaround:**

Demo: https://jsfiddle.net/BlackLabel/9x3qwfkg/

Plugin:
```js

(function(H) {
  H.wrap(
    H._modules['Extensions/ScrollablePlotArea.js'].prototype,
    'moveFixedElements',
    function(p, ...args) {
      const {
        container,
        inverted,
        scrollablePixelsX,
        scrollablePixelsY,
      } = this.chart,
        fixedRenderer = this.fixedRenderer;


      p.apply(this, args);

      [].forEach.call(
        container.querySelectorAll('.highcharts-range-selector-group'),
        (elem) => {
          (
            elem.namespaceURI === fixedRenderer.SVG_NS ?
            fixedRenderer.box :
            fixedRenderer.box.parentNode
          ).appendChild(elem);
          elem.style.pointerEvents = 'auto';
        }
      );
    }
  );
})(Highcharts);
```

*Internal note:*
Add `highcharts-range-selector-group` to list of fixed elements in `moveFixedElements()` method

**Comment by nurbek-mirai** [comment]
Hi @pawelfus,
Thanks for the workaround maybe I should have been more specific but if the `rangeSelector` is dropdown meaning if you set `dropdown` option to `always` the dropdown content doesn't show up. Any idea?

Demo: https://jsfiddle.net/0r8gamj5/1/

**Comment by pawelfus** [comment]
I think this is a separate issue, unrelated to the `scrollablePlotArea`: https://jsfiddle.net/BlackLabel/257mobyL/ 

@bm64 - could you bisect and create a separate regression issue for it?

**Comment by TorsteinHonsi** [comment]
> Add highcharts-range-selector-group to list of fixed elements in moveFixedElements() method

@pawelfus In addition to that, I think we should move that list to a static property on the ScrollablePlotArea class, so that it will be easier for us to provide customizations and workarounds in the future.

**Comment by nurbek-mirai** [comment]
Thanks guys for the fix, 
Does anyone happen to know when the new fixed version will be released?

**Comment by karolkolodziej** [comment]
@nurbek-mirai We are aiming to release a new version within the next week.

**Comment by djpysu** [comment]
News on this one?

**Comment by karolkolodziej** [comment]
It was fixed and released in version 11.4.2 see: https://www.highcharts.com/changelog/#highcharts-gantt-v11.4.2
Please use the latest 😉
